network/test_utils.go

```diff
@@ -0,0 +1,13 @@
+// +build !prod
```

Comment:
Does this exclude this file from a prod build? Where is this documented?

Reply:
[This](https://www.digitalocean.com/community/tutorials/customizing-go-binaries-with-build-tags) is a long article. tl;dr I need to add it to our makefile.